purrplingcat/PurrplingMod
# Idle mode: Animate behavior

## Types of changes

- [x] Content updates (new dialogues added or some string edited, changed data definitions and etc)
- [ ] **Merge ASAP**
- [ ] Merge before *enumerated issues or other PRs numbers*
- [x] Merge after #29 

## Description
- Added base logic of iddle mode (Iddle controller)
- Added animation behaviour for iddle controller

## How to test
Just wait, do nothing and idle annimation will be played on NPC. Check it for more NPCs

## Checklist

**Please check if the PR fulfills these requirements**
- [ ] Changes was tested and passed

## Other information
---

Closes #32, closes #34
